src/core/Global.ttslua

```diff
@@ -952,3 +957,26 @@ function onClick_defaultSettings()
   -- update UI
   updateOptionPanelState()
 end
+
+-- splash scenario title on setup
+titleSplashId = nil
```

Comment:
Would add `local` to the variable declaration and move it to the top of the script.
How about including `Loop` in the name so it's not going to be confused with the XML id?